pkg/api/controller_test.go

```diff
@@ -2017,6 +2017,52 @@ func TestController_DiffRefsHandler(t *testing.T) {
 			t.Fatalf("wrong diff type: %s", results[0].Type)
 		}
 	})
+
+	t.Run("diff refs with metadata", func(t *testing.T) {
```

Comment:
maybe on another test or one that check the default diff, check we are not getting Right set with any value.